### PR TITLE
Design of registration flow

### LINKED ISSUE #75: apply design to company details flow

### PR DESCRIPTION
**Pull request by IAfanasovMob**
- cleanup of style files
- Address validation logic
- deleted unused dependency

Closes #75

Enterpreneur-flow

### CODE REVIEW

**Comment by IAfanasovMob** [comment]
hm... my recording tool changing the colors. the app looks a bit more bright

File: src/views/Entrepreneur.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="questions">
+  <div id="questions" class="d-flex flex-column flex-fill">
```
Comment (cezaryskura) [inline src/views/Entrepreneur.vue:2]: do we still need id ?

File: src/components/entrepreneur-flow/EntrepreneurFlowIntro.vue [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+  <div class="d-flex flex-column align-items-center flex-fill">
+    <h2>Gelukt!</h2>
 
     <p>
       Je e-mail staat bij ons genoteerd. Vul de rest van je gegevens in om je
       Tegoedjes te kunnen ontvangen.
     </p>
 
-    <button class="big-red-button" type="button" @click="emitFlowNavigate"
+    <div class="flex-fill"></div>
```
Comment (cezaryskura) [inline src/components/entrepreneur-flow/EntrepreneurFlowIntro.vue:10]: why do we have this 'empty' div ?

File: src/views/Entrepreneur.vue (hunk 1 above)
Comment (IAfanasovMob) [reply src/views/Entrepreneur.vue:2]: nope. I will delete it

File: src/components/entrepreneur-flow/EntrepreneurFlowIntro.vue (hunk 2 above)
Comment (IAfanasovMob) [reply src/components/entrepreneur-flow/EntrepreneurFlowIntro.vue:10]: `flex-fill` makes this div fill in the empty space and move the button below to the bottom of the screen

**Pull request merged**